C/Chafa/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+using BinaryBuilder
+
+name = "Chafa"
+version = v"1.4.1"
+
+sources = [
+    ArchiveSource("https://hpjansson.org/chafa/releases/chafa-$(version).tar.xz",
+                  "46d34034f4c96d120e0639f87a26590427cc29e95fe5489e903a48ec96402ba3"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/chafa-*/
+
```

Comment:
```suggestion

if [[ ${target} == *darwin* ]]; then
    # For some reason building with Clang for macOS doesn't work
    export CC=gcc
fi
```

Reply:
Is this what you had in mind, @giordano?

Reply:
Yes, I think the build system is quite messy, the use of `AC_TRY_RUN` to detect the availability of functions in the compiler speaks for itself.  Maybe GCC here works just by chance, but there isn't a fundamental reason why Clang shouldn't work